pronto/api/entries.py

```diff
@@ -271,34 +271,42 @@ def get_recent_entries():
           WHERE MC.STATUS = 'Y'
           GROUP BY EM.ENTRY_AC
         ) MC ON E.ENTRY_AC = MC.ENTRY_AC
+        LEFT OUTER JOIN (
+          SELECT ENTRY_AC, 
+          LISTAGG(METHOD_AC, ',') WITHIN GROUP (ORDER BY METHOD_AC) si
+          FROM INTERPRO.ENTRY2METHOD E2M
+          GROUP BY ENTRY_AC
+        ) SIGN ON SIGN.ENTRY_AC=E.ENTRY_AC
         WHERE A.TIMESTAMP >= :1
         ORDER BY E.ENTRY_AC DESC
         """,
         (date,),
     )
     entries = []
+    curators = set()
+
     for row in cur:
-        db_user = row[4]
-        user_name = row[5]
+        user_name = row[4]
+
+        curators.add(user_name)
 
         entries.append(
             {
                 "accession": row[0],
                 "type": row[1],
                 "short_name": row[2],
+                "signatures": row[8].split(',') if row[8] else None,
```

Comment:
Keep the type of `signatures` consistent (always a `list`), and sort accessions if any:

```suggestion
                "signatures": sorted(row[8].split(',')) if row[8] else [],
```